.gitignore

```diff
@@ -58,6 +58,7 @@ typings/
 .env
 
 build/
+prebuilds/
```

Comment:
This was on purpose not on the .gitignore, as otherwise npm package will also ignore it when uploading to npm.

Thus please revert